src/it/tita/pom.xml

```diff
@@ -6,7 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-json-module</artifactId>
-    <version>2.5.0-SNAPSHOT</version>
+    <version>2.5.0</version>
```

Comment:
As I understand it, this Tita project has nothing to do with the project you are testing, so it does not require the same name or the same version number. To avoid confusion (both on the developer side and on the pipeline side) I would use another name and version SNAPSHOT. For example:
```
    <groupId>org.mule.modules</groupId>
    <artifactId>mule-json-module-it</artifactId>
    <version>1.0.0-SNAPSHOT</version>
```
What do you think?

Reply:
I think it's a good idea to separate the version numbering of the integration test project so that it doesn't cause confusion as to whether or not it should be kept in sync with JSON Module.